js/impact.js

```diff
@@ -1629,14 +1629,135 @@ var GLPIImpact = {
         this.initToolbar();
     },
 
+    popperFactory: function(ref, content, opts) {
+
+        // see https://floating-ui.com/docs/computePosition#options
+        const popperOptions = {
+            // matching the default behaviour from Popper@2
+            // https://floating-ui.com/docs/migration#configure-middleware
+            middleware: [
+                FloatingUIDOM.flip(),
```

Comment:
One the build issue is resolved, you will need to add this at the top of the script to fix the lint issue:
/* global FloatingUIDOM */

Reply:
Thanks for your help